pipeline_dp/aggregate_params.py

```diff
@@ -49,3 +50,32 @@ class AggregateParams:
     high: float = None
     budget_weight: float = 1
     public_partitions: Any = None
+
+
+@dataclass
+class SumParams:
+    """Specifies parameters for sum calculation from PrivateRDD.
+
+    Args:
+        max_partitions_contributed: Bounds the number of partitions in which one
+            unit of privacy (e.g., a user) can participate.
+        max_contributions_per_partition: Bounds the number of times one unit of
+            privacy (e.g. a user) can contribute to a partition.
+        low: Lower bound on a value contributed by a unit of privacy in a partition.
+        high: Upper bound on a value contributed by a unit of privacy in a
+            partition.
+        public_partitions: A collection of partition keys that will be present in
+            the result.
+        partition_extractor: A function for partition id extraction from an RDD record.
+        value_extractor: A function for extraction of value
+            for which the sum will be calculated.
+  """
+
+    max_partitions_contributed: int
```

Comment:
Recently it was introduced a new parameter [noise_kind](https://github.com/OpenMined/PipelineDP/blob/545166054b8ac1750dafa752165323b08ac7ee8e/pipeline_dp/aggregate_params.py#L58).

Please add it here and in creating `AggregateParams`

Please use `pipeline_dp.NoiseKind.LAPLACE` for tests